### PR TITLE
Updating releases.json of additional property

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
Updating releases.json of additional property. Needed for implementation of [this](https://github.com/trezor/trezor-suite-private/issues/104) 

To run check to validate the revisions are correct: `./packages/connect-common/scripts/check-firmware-revisions.sh`

image

### CODE REVIEW

File: packages/connect-common/scripts/check-firmware-revisions.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    echo "$REPO_DIR_NAME directory does not exist"
+    git clone https://github.com/trezor/trezor-firmware.git $REPO_DIR_NAME
+fi
+
+cd $REPO_DIR_NAME
+git fetch origin
+git checkout "$BRANCH"
+git reset "origin/$BRANCH" --hard
+
+DATA=`cat $PARENT_PATH/../files/firmware/t1b1/releases.json | jq -r '.[] | .version |= join(".") | .firmware_revision + "%" + .version'`
```
Comment (peter-sanderson) [inline packages/connect-common/scripts/check-firmware-revisions.sh]: I am, not sure if `jq` would be available in github bash env. 

- [x] Todo: check, solve with somebody (adminkos?)

Comment (peter-sanderson) [reply packages/connect-common/scripts/check-firmware-revisions.sh]: it works

File: .github/workflows/check-code-validation.yml [hunk 2]
```diff
@@ -118,6 +118,8 @@ jobs:
         uses: ./.github/actions/nx-checkout
       - name: "Minimal yarn install"
         uses: ./.github/actions/minimal-yarn-install
+      - name: Check releases.json revisions
+        run: yarn workspace @trezor/connect-common validate-releases.json
```
Comment (peter-sanderson) [inline .github/workflows/check-code-validation.yml]: todo: run only if release.json changes:

```
git diff --name-only be519ebadfe30cc73247742ef254256786232793 29a03dfd282a5a192dd94c2c38211c90e7414ede | cat
```

Comment (peter-sanderson) [reply .github/workflows/check-code-validation.yml]: `git diff --name-only origin/develop HEAD | cat`

`$BASE_BRANCH_NAME` for `origin/develop`

Comment (mroz22) [reply .github/workflows/check-code-validation.yml]: another option is to move it to https://github.com/trezor/trezor-suite/blob/fix/transport-usb-abort/.github/workflows/test-misc.yml#L1

Comment (vdovhanych) [reply .github/workflows/check-code-validation.yml]: we can use changed files action to conditionally run this - https://github.com/dorny/paths-filter

```suggestion
- uses: dorny/paths-filter@v3
  id: changeds
  with:
    filters: |
      releases-json:
        - 'packages/connect-common/files/firmware/t1b1/releases.json'

  # run only if some file in 'src' folder was changed
  - if: steps.changes.outputs.releases-json == 'true'
  - name: Check releases.json revisions
    run: yarn workspace @trezor/connect-common validate-releases.json
  ```

**Review by peter-sanderson — APPROVED** [review]

**Review by vdovhanych — APPROVED** [review]
It looks good, but i did not check the revisions.

**Pull request merged**